_activities/pyhep.md

```diff
@@ -0,0 +1,43 @@
+---
+title: PyHEP - "Python in HEP" Series of Workshops
```

Comment:
You took out too much! You should have also
```
author: "Eduardo Rodrigues"
layout: default
```
(The `layout` is really critical.)

Reply:
OK OK! (I need to set-up things as you suggest, to test updates building the site locally.)